e2e/09-price-graph-flow.e2e.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { i18nObject } from "../app/i18n/i18n-util"
+import { loadLocale } from "../app/i18n/i18n-util.sync"
+import { goBack, selector } from "./utils"
+
+describe("Price graph flow", () => {
+  loadLocale("en")
+  const LL = i18nObject("en")
+  const timeout = 30000
+
+  it("should click on price graph button", async () => {
```

Comment:
```suggestion
  it("click on price graph button", async () => {
```

the `should` is unnecessary